pkg/zonemgr/zone_file_test.go

```diff
@@ -39,18 +40,29 @@ var _ = Describe("disk zone file maintenance", func() {
 		It("should create a zone file on disk", func() {
 			testFileFunc(zoneFileContent)
 		})
+
+		It("should return nil when try to read SOA serial", func() {
+			soaSerial, err := zoneFile.ReadSoaSerial()
+			Expect(err).To(BeNil())
```

Comment:
Please consider fixing this one on either this PR or a follow one
Expect(err).ToNot(HaveOccurred())

Reply:
fixed